tests/kdc/check-iprop.in

```diff
@@ -454,9 +454,10 @@ if [ "$db_type" = lmdb ] && type mdb_stat > /dev/null 2>&1; then
     # require a sqlite3(1) shell that is capable of opening our HDB
     # files.
     echo "checking that principals in DB == entries in LMDB"
-    princs=`${kadmin} -l list '*' | wc -l`
+    # Add one to match lmdb overhead
+    princs=`(echo; ${kadmin} -l list '*') | wc -l`
     entries=`mdb_stat -n current-db.mdb | grep 'Entries:' | awk '{print $2}'`
-    [ "`expr 1 + "$princs"`" -eq "$entries" ] || exit 1
```

Comment:
The double-quotes around "$entries" are needed, we don't know that `grep` will produce any output in adverse cases.  I think that avoiding expr is simpler, but I had also considered and tested dropping quotes around $princs, and that works. If you really prefer that, I am not opposed, but I find `expr` an ugly interface, and avoid using it.